agent/lib/kontena/models/service_pod.rb

```diff
@@ -206,8 +202,11 @@ def service_host_config
           host_config['PortBindings'] = self.build_port_bindings
         end
 
-        host_config['NetworkMode'] = self.net
-        host_config['DnsSearch'] = [self.domainname, self.domainname.split('.', 2)[1]]
```

Comment:
Doesn't the service container still need the `DnsSearch`?

Reply:
I thought it will get this from infra container.. need to check this.